service/src/test/java/com/ge/predix/controller/test/ResourcePrivilegeManagementControllerIT.java

```diff
@@ -83,11 +86,15 @@ public void setup() throws Exception {
         this.testZone2 = TEST_UTILS.createTestZone("ResourceMgmtControllerIT2");
         this.zoneService.upsertZone(this.testZone);
         this.zoneService.upsertZone(this.testZone2);
+    }
+
+    @BeforeMethod
```

Comment:
Refer to comments for PolicyManagementControllerIT regarding setup() and beforeMethod().  Setting up testZone2 could be deferred until the  test that is using it - testSameResourceDifferentZones()